charts/seed-monitoring/charts/core/charts/prometheus/rules/kube-etcd3.rules.yaml

```diff
@@ -69,6 +69,51 @@ groups:
   - record: shoot:etcd_object_counts:sum_by_resource
     expr: sum(etcd_object_counts) by (resource)
 
+# etcd DB size alerts
+  - alert: KubeEtcd3MainDbSizeLimitApproaching
+    expr: (etcd_mvcc_db_total_size_in_bytes{job="kube-etcd3",role="main"} > bool 1610612736) + (etcd_mvcc_db_total_size_in_bytes{job="kube-etcd3",role="main"} <= bool 2147483648) == 2 # between 1.5GB and 2GB
+    labels:
+      service: etcd
+      severity: warning
+      type: seed
+      visibility: operator
```

Comment:
Shoot owners might be interested in receiving these alerts as well, especially since they can influence ETCD. I think we should change the visibility to all.